CompetitionTemplate/src/competition/subsystems/arm/arm_commands/DropKickstandCommand.java

```diff
@@ -0,0 +1,67 @@
+package competition.subsystems.arm.arm_commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.arm.ArmSubsystem;
+import edu.wpi.first.wpilibj.Timer;
+import xbot.common.command.BaseCommand;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class DropKickstandCommand extends BaseCommand {
+
+    ArmSubsystem arm;
+    
+    DoubleProperty kickstandDropHeight;
+    DoubleProperty kickstandRaisePower;
+    DoubleProperty kickstandRaiseDuration;
+    
+    double startTime;
+    
+    @Inject
+    public DropKickstandCommand(ArmSubsystem arm, XPropertyManager propMan) {
+        kickstandDropHeight = propMan.createPersistentProperty("KickstandDropHeight", 5.0);
+        kickstandRaisePower = propMan.createPersistentProperty("KickstandRaisePower", 0.2);
```

Comment:
We'll have to test - the arm normally is so very very strong, but it's possible all the tension in the system means we need to crank up these values.